uint16 nav_bbr_mask
uint8 reset_mode
---
bool eph
bool alm
bool health
bool klob
bool pos
bool clkd
bool osc
bool utc
bool rtc
bool aop